lektor/imagetools.py

```diff
@@ -365,15 +379,15 @@ def process_image(ctx, source_image, dst_filename, width, height=None,
     if height is not None:
         resize_key += 'x' + str(height)
 
-    cmdline = [im, source_image]
+    cmdline = [im, source_image, '-auto-orient']
     if crop:
         cmdline += ['-resize', resize_key + '^',
                     '-gravity', 'Center',
                     '-extent', resize_key]
     else:
         cmdline += ['-resize', resize_key]
 
-    cmdline += ['-auto-orient', '-quality', str(quality), dst_filename]
```

Comment:
Any particular reason why you moved the `-auto-orient` parameter earlier? It's fine either way, I'm just curious if there's some rationale that I'm missing.